### PR TITLE
[DC-2022] Files for AWS community day

### PR DESCRIPTION
**Pull request by pburkholder**

### CODE REVIEW

**Comment by MikeRosTX** [comment]
Is this ready to be merged? 
Because it is in **DRAFT** mode, and we cannot merge it.

**Comment by pburkholder** [comment]
I'm still making sure the new sponsor is correctly showing up...

File: data/events/2022-washington-dc.yml [hunk 1]
```diff
@@ -236,6 +238,8 @@ sponsors:
     level: community
   - id: kubeevents
     level: community
+  - id: aws-community-day-dmv
```
Comment (glasnt) [inline data/events/2022-washington-dc.yml:241]:
```suggestion
  - id: awscommunityday-dmv
```

To match the data and image name

**Comment by pburkholder** [comment]
Please merge this monstrosity, @mattstratton , and I'll fix the logo on the next iteration. I think this is my first time the # of commits exceeds the # lines of code, and I want it over with.

**Comment by mattstratton** [comment]
Lol yeah it isn't a blocker, just a observation. I'll merge!

**Pull request merged**